internetarchive/session.py

```diff
@@ -34,7 +34,7 @@
 import platform
 import sys
 import warnings
-from typing import Iterable, Mapping, MutableMapping
+from typing import Iterable, Mapping, MutableMapping, Union, cast
```

Comment:
Neither `Union` nor `cast` are used below (anymore), so they shouldn't be imported either.

Reply:
Enable the appropriate flake8 is a separate pr.